src/cli/test/parse-flags.spec.ts

```diff
@@ -30,556 +20,216 @@ describe('parseFlags', () => {
-    ['--max-workers', '4'],
-    ['--maxWorkers', '4e+0'],
-    ['--maxWorkers', '40e-1'],
-  ])('should parse %p, %p', (...args) => {
-    const flags = parseFlags(args, sys);
-    expect(flags.maxWorkers).toBe(4);
-  });
-
-  it('should parse --maxWorkers 4', () => {
-    args[0] = '--maxWorkers';
-    args[1] = '4';
-    const flags = parseFlags(args, sys);
-    expect(flags.maxWorkers).toBe(4);
-  });
-
-  it('should parse --maxWorkers=4', () => {
-    args[0] = '--maxWorkers=4';
-    const flags = parseFlags(args, sys);
-    expect(flags.maxWorkers).toBe(4);
-  });
-
-  it('should parse --max-workers 4', () => {
-    args[0] = '--max-workers';
-    args[1] = '4';
-    const flags = parseFlags(args, sys);
-    expect(flags.maxWorkers).toBe(4);
-  });
-
-  it('should parse --maxWorkers=50%', function () {
-    // see https://jestjs.io/docs/27.x/cli#--maxworkersnumstring
-    const flags = parseFlags(['--maxWorkers=50%']);
-    expect(flags.maxWorkers).toBe('50%');
-  });
-
-  it('should parse --max-workers=1', () => {
-    args[0] = '--max-workers=1';
-    const flags = parseFlags(args, sys);
-    expect(flags.maxWorkers).toBe(1);
-  });
-
-  it('should not parse --max-workers', () => {
-    const flags = parseFlags(args, sys);
-    expect(flags.maxWorkers).toBe(null);
-  });
-
-  it('should parse --no-open', () => {
-    args[0] = '--no-open';
-    const flags = parseFlags(args, sys);
-    expect(flags.open).toBe(false);
-  });
-
-  it('should parse --port', () => {
-    args[0] = '--port';
-    args[1] = '8888';
-    const flags = parseFlags(args, sys);
-    expect(flags.port).toBe(8888);
-  });
-
-  it('should parse -p', () => {
-    args[0] = '-p';
-    args[1] = '4444';
-    const flags = parseFlags(args, sys);
-    expect(flags.port).toBe(4444);
-  });
-
-  it('should parse --prod', () => {
-    args[0] = '--prod';
-    const flags = parseFlags(args, sys);
-    expect(flags.prod).toBe(true);
-  });
-
-  it('should parse --profile', () => {
-    args[0] = '--profile';
-    const flags = parseFlags(args, sys);
-    expect(flags.profile).toBe(true);
-  });
-
-  it('should parse --prerender', () => {
-    args[0] = '--prerender';
-    const flags = parseFlags(args, sys);
-    expect(flags.prerender).toBe(true);
-  });
-
-  it('should parse --root', () => {
-    args[0] = '--root';
-    args[1] = 'custom-www';
-    const flags = parseFlags(args, sys);
-    expect(flags.root).toBe('custom-www');
-  });
-
-  it('should parse --screenshot', () => {
-    args[0] = '--screenshot';
-    const flags = parseFlags(args, sys);
-    expect(flags.screenshot).toBe(true);
-  });
-
-  it('should parse --screenshot-connector scripts/connector.js', () => {
-    args[0] = '--screenshot-connector';
-    args[1] = 'scripts/connector.js';
-    const flags = parseFlags(args, sys);
-    expect(flags.screenshotConnector).toBe('scripts/connector.js');
-  });
+  describe.each<LogLevel>(['info', 'warn', 'error', 'debug'])('logLevel %s', (level) => {
```

Comment:
I pulled the `LogLevel` specs together here basically just for clarity and organization